packages/kolibri-common/strings/bulkUserManagementStrings.js

```diff
@@ -0,0 +1,235 @@
+    context: 'Informational message about trash deletion timeline and undo option',
+  },
+  usersTrashedNotice: {
+    message: 'Selected users have been moved to trash',
+    context: 'Brief notification confirming users were moved to trash',
+  },
+  trashUndoneNotice: {
+    message: 'Move to trash has been undone',
+    context: 'Notification confirming that the trash action was reversed',
+  },
+
+  // Copy a class
+  copyClass: {
+    message: 'Copy class',
+    context: 'Page heading and label for confirmation button',
+  },
+  coachesAssignedToClass: {
+    message: 'Coaches assigned to this class',
+    context: 'Heading for table listing coaches',
+  },
+  numCoachesSelected: {
+    message: '{n, number} {n, plural, one {coach} other {coaches}} selected',
+    context: "Label showing the user how many coaches they've selected",
+  },
+  classCopiedSuccessfully: {
+    message: 'Class copied successfully',
+    context: 'Message shown when class copying succeeds',
+  },
+
+  // User Creation
+  newUsers: {
+    message: 'New users',
+    context: 'Title of page listing recently created users',
+  },
+  backToUsers: {
+    message: 'Back to Users',
+    context: 'Link leading back to regular users table, away from the user creation page',
+  },
+
+  // User Creation Modal
+  newUsersCreatedSuccess: {
+    message: '{n, number} new {n, plural, one {user} other {users}} successfully created!',
+    context: 'Title of modal shown on successfull creation of users',
+  },
+  newUsersModalMessage: {
+    message: 'You have successfully created {n, number} new {n, plural, one {user} other {users}}.',
+    context: 'Initial message body in success modal when users are created',
+  },
+  whatsNext: {
+    message: "Here's what you can do next:",
+    context: 'Message preceding a list of things the user can do now that they added users',
+  },
+  carryOutBulkActions: {
+    message: 'Carry out bulk actions',
+    context: 'One of the options explained to the user',
+  },
+  carryOutBulkActionsExplainer: {
+    message: 'Like enrolling your entire selection in one or more classes',
+  },
+  saveAndAddAnother: {
+    message: 'Save and add another',
+    context:
+      'Button label on user creation form that submits the current form and then clears it for another entry on success',
+  },
+  saveAndContinue: {
+    message: 'Save and continue',
+    context:
+      'Action label for users to save a new user and make another without leaving the current form',
+  },
+  saveAndContinueExplainer: {
+    message:
+      'You can close this modal interface and all your newly added users willb e saved to this facility in the Users page',
+    context: 'Explaining what the user can do if they save and continue',
+  },
+  enrollInClass: {
+    message: 'Enroll in class',
+    context: 'Action button label for beginning bulk actions',
+  },
+  copyOfClass: {
+    // is this valid ICU syntax for text input
+    message: 'Copy of {class}',
```

Comment:
Because we typically '' wrap user generated text, this should be 
`Copy of '{class}'`